oss_fuzz_integration/oss-fuzz-patches.diff

```diff
@@ -61,9 +61,13 @@ index c934d3b5..d6844e63 100755
 +  find $SRC/ -name "*.data.yaml" -exec cp {} $SRC/inspector-tmp/ \;
 +
 +  # Move coverage report in case
-+  if [ -d "$OUT/fuzzer_stats" ]
-+  then
-+    cp $OUT/fuzzer_stats/*.covreport $SRC/inspector-tmp/
++  if [[ -z ${CLOUD_BUILD_ENV:+dummy} ]]; then
++     if [ -d "$OUT/fuzzer_stats" ]
++     then
++        cp $OUT/fuzzer_stats/*.covreport $SRC/inspector-tmp/
++     fi
++  else
++     find /workspace/ -name "*.covreport" -exec cp {} $SRC/inspector-tmp/ \;
 +  fi
 +  ls -la $SRC/inspector-tmp
```

Comment:
We may as well remove this line I guess - we don't really use it. It was mainly for debugging back in the day. If you find it useful feel free to leave it

Reply:
Will squash and merge - the `ls` command is okay to have for now just for quick debugging help.